x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/related_dashboards/dashboard_tile.tsx

```diff
@@ -49,12 +50,19 @@ export function DashboardTile({
     <>
       <EuiFlexGroup gutterSize="xs" responsive={false} key={dashboard.id}>
         <EuiFlexItem key={dashboard.id}>
+          {/* Allowing both href and onClick to allow telemetry to be reported */}
+          {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
           <EuiLink
             data-test-subj="o11yDashboardTileLink"
             href={dashboardLocator?.getRedirectUrl({
               dashboardId: dashboard.id,
             })}
             target="_blank"
+            onClick={() => {
+              if (telemetryClient) {
+                telemetryClient.reportLinkedDashboardViewed(dashboard.id);
```

Comment:
I wanted to ship the rule type with the dashboard ID (for whatever benefits we can get), but it's not available. We need to pass it explicitly from two levels up only for telemetry reasons, which is not worth it.

Reply:
can we use `actionButtonProps.ruleType` for rule type? I think dashboard ID is not needed. We want to capture how many times users clicked on linked and suggested dashboards and how many were unique users. It's less important if users clicked on same dashboard since it depends on how they have setup dashboards - some dashboards might be more frequently used than others. For understanding how our features are used, it's not needed.